ghtt/assignment.py

```diff
@@ -93,12 +95,29 @@ def create_pr(ctx, branch, title, body, source, students=None, groups=None, bran
             command = ["git", "push", g_repo.ssh_url, "master:{}".format(branch)]
             cwd = source
             print("\nwill run `{}`\nin directory `{}`.".format(command, cwd))
-            if click.confirm('Do you want to continue?'):
+            if click.confirm('Do you want to continue?', default=False, abort=True):
                 if not branch_already_pushed:
                     subprocess.check_call(command, cwd=cwd)
                 pr = g_repo.create_pull(title, "master", branch, body=body)
                 click.secho("created pull request {}".format(pr.html_url))
         else:
+            if not auto_mode:
+                user_choice = click.prompt('Do you want to create the PR in {}?'.format(repo.name),
```

Comment:
Why is this only asked when the branch is already pushed? Shouldn't this be asked regardless? I think it's useful for both scenarios. For example, to check the first PR after it's created and then continue with the rest after you checked.